t/t7900-maintenance.sh

```diff
@@ -1011,4 +1011,34 @@ test_expect_success 'repacking loose objects is quiet' '
 	)
 '
 
+test_expect_success 'vfs-cache-move task' '
+	#test_when_finished "rm -rf repo" &&
+	git init repo &&
+	(
+		cd repo &&
+
+		test_commit something &&
+		git config set gvfs.sharedcache ../cache &&
+		git config set maintenance.gc.enabled false &&
+		git config set maintenance.vfs-cache-move.enabled true &&
+		git config set maintenance.vfs-cache-move.auto 1 &&
+
+		touch .git/objects/pack/vfs-12345678.pack &&
+		touch .git/objects/pack/vfs-12345678.keep &&
+		touch .git/objects/pack/vfs-12345678.rev &&
+		touch .git/objects/pack/vfs-12345678.idx &&
```

Comment:
I fear that we have to make this a proper `.pack` and `.idx`; This is relatively easy, though:

```sh
printf 'blob\ndata <<END\n%s\nEND\n\n' 1 2 3 4 5 |
git -c fastimport.unpackLimit=0 fast-import
```

The only non-obvious thing is to find out what the pack file is called, as `fast-import` does not report _that_ ;-)